apis/v1alpha1/convert.go

```diff
@@ -7,7 +7,7 @@ import (
 	"errors"
 	"fmt"
 
-	"gopkg.in/yaml.v3"
+	go_yaml "github.com/goccy/go-yaml"
```

Comment:
(nit)
```suggestion
	yaml "github.com/goccy/go-yaml"
```

Reply:
I think it's better for this to consistently be `go_yaml` everywhere. When I see `yaml`, I assume it's `gopkg.in/yaml`, so a different alias seems useful to indicate that it's not.